src/client/pps/pps.proto

```diff
@@ -110,6 +106,28 @@ message ParallelismSpec {
   double coefficient = 3;
 }
 
+message ObjectStore {
+  string URL = 1;
+}
+
+message SQLDb {
+  string driver = 1;
+  string URL = 2;
+  repeated string init = 3;
+  repeated string tables = 4;
+}
+
+enum ConnectorType {
+  OBJECT_STORE = 0;
+  SQL_DB = 1;
+}
+
+message Connector {
+    ConnectorType type = 1;
```

Comment:
* I think that returning an error is the right thing to do here (which I think means I agree...)

* Nope, not really. I'll submit a PR to start ignoring the strategy field.